### ISSUE TITLE
Tool table wiodget: expand for column "rotation"

### ISSUE THREAD

**Issue by hansu** [LinuxCNC/linuxcnc#3064]
![grafik](https://github.com/user-attachments/assets/68ee4fe6-a717-444d-85d1-67871a4f873e)

**Comment by Sigma1912** [comment]
I agree. This makes much more sense as each WCS has it's own 'R' value.

**Comment by andypugh** [comment]
And having a spurious rotation is an occasional cause of trouble. (I saw a recent forum post where someone said that their X and Y axes had swapped. That was the cause.

**Comment by c-morley** [comment]
Does each system have it's own rotation? I thought the rotation was was separate and added after the user system.

**Comment by Sigma1912** [comment]
Yes, each system has it's own rotation. AxisGUI displays R values correctly for each system.

**Comment by hansu** [comment]
Yes, except G92. See https://linuxcnc.org/docs/devel/html/gcode/g-code.html#gcode:g10-l2

**Comment by c-morley** [comment]
What python attribute gives the rotation setting for systems not in use?
rotation_xy only gives the current rotation.

**Comment by Sigma1912** [comment]
As far as I'm aware there are only python attributes for for the currently active WCS system.  I think one would have to query parameters #5230 + i*20, i=[0..8]

**Comment by c-morley** [comment]
Ahh yes ,it's coming back to me now. We read the var file directly for the other user systems. I assume there is a var in that file for the other rotations too.

**Comment by hansu** [comment]
@c-morley Are you currently working on this? If yes - I thought it would be nice if we could use the calculator widget to enter the offset values. What do you think?

**Comment by hansu** [comment]
Fixed by https://github.com/LinuxCNC/linuxcnc/pull/3457
